client/extensions/wp-super-cache/style.scss

```diff
@@ -237,6 +237,12 @@
 	white-space: pre;
 }
 
+// Plugins Tab
+
+.wp-super-cache__plugins-tab .form-setting-explanation {
+	margin-left: 36px;
```

Comment:
Could we also add 5px of bottom margin here? It's a wee bit too close to the next toggle option.